diff --git a/menu/menu_entry.c b/menu/menu_entry.c
index 2ddc723..ad140e3 100644
--- a/menu/menu_entry.c
+++ b/menu/menu_entry.c
@@ -493,7 +493,7 @@ int menu_entry_action(menu_entry_t *entry, unsigned i, enum menu_action action)
                   entry->label, entry->type, i);
          break;
       case MENU_ACTION_SEARCH:
-         menu_input_dialog_start_search();
+         //menu_input_dialog_start_search();
          break;

       case MENU_ACTION_SCAN:
